### PR TITLE
Bump sailor version to 2.2.1.

### PR DESCRIPTION
**Pull request by jhorbulyk**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -16,7 +16,7 @@
         "npm": ">=5.4"
     },
     "dependencies": {
-        "elasticio-sailor-nodejs": "2.2.0",
+        "elasticio-sailor-nodejs": "2.2.1",
```
Comment (zubairov) [inline package.json]: 2.2.x?

**Comment by zubairov** [comment]
@jhorbulyk Can you also do a ``0.0.x`` for the ``elasticio-node``

File: package.json (hunk 1 above)
Comment (jhorbulyk) [reply package.json]: See @drobiazko 's thoughts on why that would be a bad idea: https://elasticio.slack.com/archives/G6DRG6HEW/p1507124793000172

File: package.json [hunk 2]
```diff
@@ -16,7 +16,7 @@
...
         "elasticio-node": "0.0.7",
```
Comment (Nazar910) [inline package.json]: can you bump version of elasticio-node to 0.0.8? (just in case) 0.0.7 has some conflicts with sailor 2.2.0.

**Pull request merged**